plugins/runtime/threads/thread-dump-modal-.component.less

```diff
@@ -0,0 +1,3 @@
+.thread-dump-modal-output{
+  height:80vh;
```

Comment:
also, don't forget to add space ' ' between name and value. > `height: 80vh;`
take care of looking more.